### ISSUE TITLE
Neutral mobs not spawning anymore

### ISSUE THREAD

**Issue by h2owasser** [ItsBlackGear/CaveBiomeAPI#16]
Noticed something on Forge 37.0.103:

With this mod installed neutral mobs like sheep, cows, pigs simply do not spawn (naturally) in the world anymore.

Uninstalled all other mods and tested it, too. (I know does not make much sense when you use this mod as bridge...but i needed to narrow it down a little)

Hope this bug report helps

**Comment by ItsBlackGear** [comment]
Hold on, what version of the mod are you using currently?

**Comment by h2owasser** [comment]
"Latest" Version on Curseforge cavebiomeapi-1.17.1-1.4.0.jar from 10/07/2021

**Comment by ItsBlackGear** [comment]
Thanks, i'll check right away!

**Comment by ItsBlackGear** [comment]
Well, i created a new world and took me a while to find pigs, but they
spawned right in front of me, but don't worry

I'll do a tweak so they can spawn as they should c: thanks for your report!

**Comment by h2owasser** [comment]
In villages for example i found some, too.

In the wilderness in comparison to a vanilla world they were quite rare (i think i found 1 group in around 4000 blocks).

Thank you for the quick response.

**Comment by ItsBlackGear** [comment]
Yeah they are! They can spawn randomly but not as often as they should
